### DISCUSSION TITLE
Is there a plan to release 2.1.5 anytime soon?

### DISCUSSION THREAD

**Discussion by sunilvaltix** [openzfs/zfs#13536] [General]
I see several PRs have landed in 2.1.5 branch, including support for 5.18. Is there a plan to release 2.1.5 anytime soon?

What are the current showstoppers for a 2.1.5 release?

**Comment by tonyhutter** [comment]
Yes, we are putting together a 2.1.5 release right now: https://github.com/openzfs/zfs/pull/13532

**Comment by sunilvaltix** [comment]
Thanks for the update! Really appreciate it!

**Comment by apateluk** [comment]
This is great to hear thanks.

I foolishly upgraded by Ubuntu 22.04LTS Kernel from 5.15 to 5.18 (for other reasons) not realising it would break ZFS support. Being new to Linux, what steps would I need to take once the 2.1.5 release is made ? For example, will this get adopted into the Ubuntu repo ?

Thanks

**Comment by apateluk** [comment]
Should this release work ok with v5.18.4 ?
